app/components/selection/handle.element.js

```diff
@@ -0,0 +1,184 @@
+import $ from 'blingblingjs'
+import { HandleStyles } from '../styles.store'
+
+export class Handle extends HTMLElement {
+
+	constructor() {
+		super()
+    this.$shadow = this.attachShadow({mode: 'closed'})
+    this.styles = [HandleStyles]
+	}
+
+  connectedCallback() {
+    this.$shadow.adoptedStyleSheets = this.styles
+		this.$shadow.innerHTML = this.render();
```

Comment:
```suggestion
		this.$shadow.innerHTML = this.render()
```
😅

Reply:
Fixed in https://github.com/GoogleChromeLabs/ProjectVisBug/pull/569/commits/b1c9084dcbe8b49b5537050a10a9ee1daa75f74f. I also noticed that the project uses spaces everywhere, whereas my IDE default is tabs so fixed that too.

Might help to set up prettier

Reply:
theres an editor config, but not everyone has that plugin installed